images/ad-server/Containerfile

```diff
@@ -1,7 +1,10 @@
 FROM registry.fedoraproject.org/fedora:34
 
-RUN dnf install -y samba-dc procps-ng /usr/bin/smbclient
-RUN rm -f /etc/samba/smb.conf
+ARG INSTALL_PACKAGES_FROM=default
```

Comment:
Even if we don't do much with it, we should probably also have a "SAMBA_SPECIFICS" arg here like we do in the server image as a "breadcrumb"

Reply:
Fine.